sdk/core/Azure.Core.TestFramework/src/TestEnvironment.cs

```diff
@@ -131,7 +131,7 @@ static TestEnvironment()
         /// <summary>
         ///   The URL of the Azure Authority host to be used for authentication. Recorded.
         /// </summary>
-        public string AuthorityHostUrl => GetRecordedOptionalVariable("AZURE_AUTHORITY_HOST");
+        public string AuthorityHostUrl => GetRecordedOptionalVariable("AZURE_AUTHORITY_HOST") ?? "https://login.microsoftonline.com/";
```

Comment:
If I recall, @pakrym had some thoughts about including this as a lone default here...

Reply:
The resource deployment script should provide this value: https://github.com/Azure/azure-sdk-tools/blob/154d977720f56e6816fb99e1e6ff6970321dda8a/eng/common/TestResources/New-TestResources.ps1#L492 we shouldn't hard code it here.